packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -390,7 +396,10 @@ DarwinArch getIOSArchForName(String arch) {
   return null;
 }
 
-String getNameForTargetPlatform(TargetPlatform platform) {
+String getNameForTargetPlatform(TargetPlatform platform, {DarwinArch darwinArch}) {
+  if (darwinArch != null) {
```

Comment:
I'd move this to a nested switch under the  `TargetPlatform.ios` case. I think that's the only case where it means anything.

Reply:
Done